setup.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from setuptools import setup
+
+packages = \
+['vosekast_control', 'vosekast_control.connectors', 'vosekast_control.utils']
+
+package_data = \
+{'': ['*']}
+
+install_requires = \
+['gmqtt>=0.6.2,<0.7.0', 'pendulum>=2.0.5,<3.0.0', 'pyserial>=3.4,<4.0']
+
+extras_require = \
+{':platform_machine != "armv7l"': ['rpi @ '
+                                   'git+https://github.com/nosix/raspberry-gpio-emulator/@master'],
+ ':platform_machine == "armv7l"': ['RPi.GPIO>=0.7.0,<0.8.0'],
+ ':sys_platform == "linux"': ['uvloop>=0.14.0,<0.15.0']}
```

Comment:
where belongs that setup.py? Until now we dont need that.

Reply:
the file was created automatically

Reply:
I think you can remove this file.

Reply:
done